_specs/ecip-11355.md

```diff
@@ -0,0 +1,207 @@
+---
+ecip: 11355
+title: MESS (Modified Exponential Subjective Scoring)
+lang: en
+author: Isaac <b5c6@protonmail.com>
+discussions-to: TODO
```

Comment:
maybe create a ticket here on this repository?

Reply:
https://github.com/ethereumclassic/ECIPs/pull/373/commits/b1984548ea34c1c692784b04ec5edadc26149258